### DISCUSSION TITLE
part-ii/2001-84/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#290] [Question comments]
Part II 2001 Representation Theory: B1.6 

https://questions.tripos.org/part-ii/2001-84/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The final table is on the second page of this document (it also has the steps to find the solution). http://www.rnta.eu/Yogyakarta2020/chartable.pdf

</details>
